src/main/java/uk/gov/ch/developer/docs/config/NavBarConfig.java

```diff
@@ -22,28 +23,28 @@
     private String devGuideURL;
 
     @Bean
-    public NavBarModel getNavBarModel() {
+    public NavBarModelBuilder getNavBarModelBuilder() {
 
-        NavBarModel model = new NavBarModel();
-        NavItemList manageApplications = model.addHeading("Manage Applications");
+        NavBarModelBuilder model = new NavBarModelBuilder();
+        NavItemList manageApplications = model
+                .addHeading("Manage Applications", EnumSet.of(DisplayRestrictions.USER_REQUIRED));
```

Comment:
Probably the building of the set should be internal to the `addHeading` using `DisplayRestrictions...` as an overload.